projects/ion/src/lib/button/button.component.ts

```diff
@@ -26,6 +31,10 @@ export class ButtonComponent {
   @Input() danger? = false;
   @Input() disabled? = false;
   @Input() loading? = false;
+  @Input() loadingMessage = 'Carregando...';
```

Comment:
You need add a test case with a default message.

Reply:
the first test case of the 'load ButtonComponent' already checks the default message. :smile: